gui/src/test/java/kcl/teamIndexZero/traffic/gui/GUI_PrimitivesTest.java

```diff
@@ -0,0 +1,48 @@
+package kcl.teamIndexZero.traffic.gui;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by kumarawijeet on 3/16/2016.
+ */
+public class GUI_PrimitivesTest {
```

Comment:
Totally understand your point now. Just the thing that don't know how to write them, have had a look at the example test cases but didn't find much. However will work on it and fix it, and create a new pull request to incorporate it, at this moment I can't really think of a way in which I can do it quickly, especially after I have just deleted the file in the recent push.                                                                                                                                                                                                                                                                                                                                        Sent from my BlackBerry 10 smartphone on the EE network.                                                                                                                                                                                                                From: Alexander CherednichenkoSent: Monday 21 March 2016 08:10To: teamIndexZero/index_zero_trafficsystemReply To: teamIndexZero/index_zero_trafficsystemCc: kumar awijeetSubject: Re: [index_zero_trafficsystem] Gui graphics kumar awijeet (#9)In gui/src/test/java/kcl/teamIndexZero/traffic/gui/GUI_PrimitivesTest.java:
> @@ -0,0 +1,48 @@
> +package kcl.teamIndexZero.traffic.gui;
> +
> +import org.junit.After;
> +import org.junit.Before;
> +import org.junit.Test;
> +
> +import static org.junit.Assert.*;
> +
> +/**
> + * Created by kumarawijeet on 3/16/2016.
> + */
> +public class GUI_PrimitivesTest {

Point of unit test is to do that not manually (one time) but to add this check into the software itself. Later, in 6 weeks, when someone will be changed your code (who knows?), in case they make it working differently, they will notice the failed tests and fix them. It is like other parts of the code may depend on your logic - exceptions thrown, correct calculations etc - but we never know if the logic is still same - so we write a test to check if the logic is still the same.

—You are receiving this because you authored the thread.Reply to this email directly or view it on GitHub